api/tests/shared/unit/domain/services/language-service_test.js

```diff
@@ -0,0 +1,36 @@
+import { LanguageNotSupportedError } from '../../../../../src/shared/domain/errors.js';
+import * as languageService from '../../../../../src/shared/domain/services/language-service.js';
+
+import { catchErrSync, expect } from '../../../../test-helper.js';
+
+describe('Unit | Shared | Domain | Services | Language Service', function () {
+  describe('#assertLanguageAvailability', function () {
+    context('when language code is not provided', function () {
+      it('returns nothing (undefined)', function () {
```

Comment:
```suggestion
      it('returns nothing ', function () {
```
Pour avoir la même description que le test en bas 😄